### ISSUE TITLE
reindex single record: electronic access fields

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/bibdata#1039]
Child of #748  - see implementation notes there

Depends on #1054
Use the info gathered in #1001 

Enrich the record by pulling from the AVE section of the API response and populating marc field 951.

This is blocked on the data coming through from the API response; inquired with exlibris https://3.basecamp.com/3765443/buckets/20717908/messages/3622978401

**Comment by hackartisan** [comment]
closing in favor of #1478
